src/main/resources/pdiresources/transformations/validate-program-workflows.ktr

```diff
@@ -444,7 +444,7 @@
     <database>merge_db</database>
```

Comment:
made some former suggestion of merge_db being read from a property too.

Reply:
Done